src/overlays/actors/ovl_Dm_Stk/z_dm_stk.c

```diff
@@ -919,23 +919,23 @@ void DmStk_PlaySfxForCutscenes(DmStk* this, PlayState* play) {
     if (play->csCtx.state != 0) {
         switch (play->sceneId) {
             case SCENE_LOST_WOODS:
-                if (gSaveContext.sceneSetupIndex == 1) {
+                if (gSaveContext.sceneLayer == 1) {
                     DmStk_PlaySfxForIntroCutsceneFirstPart(this, play);
-                } else if (gSaveContext.sceneSetupIndex == 0) {
+                } else if (gSaveContext.sceneLayer == 0) {
                     DmStk_PlaySfxForIntroCutsceneSecondPart(this, play);
-                } else if ((gSaveContext.sceneSetupIndex == 2) && (play->csCtx.currentCsIndex == 0)) {
+                } else if ((gSaveContext.sceneLayer == 2) && (play->csCtx.currentCsIndex == 0)) {
                     DmStk_PlaySfxForObtainingMajorasMaskCutscene(this, play);
                 }
                 break;
```

Comment:
On that note, how do sceneLayer's work with cutscenes? Is `gSaveContext.sceneLayer == 0` also a cutscene layer? I.e. is there an equivalent of 
`#define IS_CUTSCENE_LAYER (gSaveContext.sceneLayer >= SCENE_LAYER_CUTSCENE_FIRST)`
in MM?

Reply:
from z_play `gSaveContext.sceneSetupIndex = (gSaveContext.save.cutscene & 0xF) + 1;`
so essentially layer 0 is gameplay and >= 1 is a cutscene (and would therefore be mm's `IS_CUTSCENE_LAYER` equivalent).

Fact that there exists some leftover >= 4 from oot means they probably did not have this macro, but yea it still makes things alot more readable. not sure how often mm checks for >= 1 though

Reply:
I'm not seeing any `>= 1` but instead see lots of `gSaveContext.sceneLayer == 0` and `gSaveContext.sceneLayer != 0`, which would be the equivalent is seems. 

I commented here because sceneLayer 0 still seems to be a cutscene, but I guess this system is full of exceptions?

Reply:
i would need more context of what this actor is and what the cutscene is. But in general you can have a cutscene in layer 0 if it is for example a proximity based cutscene that you walk into it to trigger